test/proxy_configuration.test.js

```diff
@@ -206,9 +241,9 @@ describe('Apify.createProxyConfiguration()', () => {
     test('should throw group is not available', async () => {
```

Comment:
should throw when group is not available

Reply:
fixed.